docs/WALKTHROUGH.md

```diff
@@ -0,0 +1,408 @@
+# Service Catalog Demonstration Walkthrough
+
+This document outlines the basic features of the service catalog by walking
+through a basic use case.
+
+## Step 0 - Prerequisites
+
+### Starting Kubernetes with DNS
+
+You *must* have a Kubernetes cluster with cluster DNS enabled.
+
+* If you are using Google Container Engine or minikube, no additional action is
+required.
+* If you are using hack/local-up-cluster.sh, ensure to set
+KUBE_ENABLE_CLUSTER_DNS:
+
+```console
+KUBE_ENABLE_CLUSTER_DNS=true hack/local-up-cluster.sh -O
+```
+
+### Getting Helm and installing Tiller
+
+If you already have Helm v2 or newer, executing `helm init` should be all that's
+required. Detailed helm installation instructions are available
+[here](https://github.com/kubernetes/helm/blob/master/docs/install.md).
+
+## Step 1 - Installing the Service Catalog System
+
+The service catalog is conveniently packaged as a [helm](http://helm.sh/) 
+chart for installation.
+
+The chart is located in the [charts/catalog](../charts/catalog) directory in this
+repository, and supports a wide variety of customizations which are laid out in
+the README.md in that directory. To install the service-catalog with sensible
+defaults, execute this command from within the Kubernetes directory:
+
+```console
+helm install <path to service-catalog>/charts/catalog --name catalog --namespace catalog
```

Comment:
I would drop the leading `<path to service-catalog>/` since the instructions above say specifically from which directory (the project root) to execute the command. The relative path `charts/catalog` should be sufficient.